service/src/test/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderServiceTest.java

```diff
@@ -261,6 +262,7 @@ private void assertCommonC21Fields(GeneratedOrder order) {
         }
 
         expectedMap
+            .put("furtherDirections", "Example Directions")
```

Comment:
we could probably move this expected map out to a private method - I know you are just adding one line here to existing code though so it's up to you :)

Reply:
it's already within a private method, so I am just going to leave as is